src/hooks/useBalanceChartData/useBalanceChartData.ts

```diff
@@ -31,35 +31,21 @@ import {
 import { cosmosAssetId } from 'state/slices/portfolioSlice/utils'
 import {
   selectAccountSpecifiers,
+  selectCryptoPriceHistoryTimeframe,
+  selectFiatPriceHistoriesLoadingByTimeframe,
+  selectFiatPriceHistoryTimeframe,
   selectPortfolioAssets,
   selectPortfolioCryptoBalancesByAccountIdAboveThreshold,
   selectPriceHistoriesLoadingByAssetTimeframe,
-  selectPriceHistoryTimeframe,
+  selectRebasesByFilter,
   selectTotalStakingDelegationCryptoByAccountSpecifier,
   selectTxsByFilter,
 } from 'state/slices/selectors'
-import { selectRebasesByFilter } from 'state/slices/txHistorySlice/selectors'
 import { Tx } from 'state/slices/txHistorySlice/txHistorySlice'
 import { useAppSelector } from 'state/store'
 
 import { includeStakedBalance, includeTransaction } from './cosmosUtils'
 
-type PriceAtBlockTimeArgs = {
-  date: number
-  assetPriceHistoryData: HistoryData[]
-}
-
-type PriceAtBlockTime = (args: PriceAtBlockTimeArgs) => number
```

Comment:
this fn prob goes back after we index and normalize fiat data on the way into the store - selectors shouldn't be calling it